workspaces/scorecard/plugins/scorecard/src/components/Common/PermissionRequiredState.tsx

```diff
@@ -52,17 +55,17 @@ const PermissionRequiredState = () => {
               lineHeight: 1.5,
             })}
           >
-            To view Scorecard plugin, contact your administrator to give the{' '}
+            {t('permissionRequired.description')}{' '}
             <Typography
               component="span"
               sx={theme => ({
                 fontWeight: 'bold',
                 color: theme.palette.text.primary,
               })}
             >
-              scorecard.metric.read
+              {t('permissionRequired.permission')}
             </Typography>{' '}
-            permission.
+            {t('permissionRequired.descriptionSuffix')}
```

Comment:
This should be one translation that uses the permission (a fix string, no translation needed) as a param.

Something like:

```
{t('permissionRequired.description', {
  permission: (
    <Typography
      component="span"
        sx={theme => ({
          fontWeight: 'bold',
          color: theme.palette.text.primary,
       })}
    >
      scorecard.metric.read
    </Typography>
  )
})}
```

This makes it easier to translate that string.